sktime/forecasting/base/adapters/_neuralforecast.py

```diff
@@ -0,0 +1,290 @@
+# copyright: sktime developers, BSD-3-Clause License (see LICENSE file)
+"""Implements adapter for NeuralForecast models."""
+import abc
+import functools
+import typing
+
+import pandas
+
+from sktime.forecasting.base import BaseForecaster, ForecastingHorizon
+
+__all__ = ["_NeuralForecastAdapter"]
+__author__ = ["yarnabrina"]
+
+
+class _NeuralForecastAdapter(BaseForecaster):
+    """Base adapter class for NeuralForecast models.
+
+    Parameters
+    ----------
+    freq : str
+        frequency of the data, see available frequencies [1]_ from ``pandas``
+    local_scaler_type : str (default=None)
+        scaler to apply per-series to all features before fitting, which is inverted
+        after predicting
+
+        can be one of the following:
+
+        - 'standard'
+        - 'robust'
+        - 'robust-iqr'
+        - 'minmax'
+        - 'boxcox'
+    futr_exog_list : str list, (default=None)
+        future exogenous variables
+    verbose_fit : bool (default=False)
+        print processing steps during fit
+    verbose_predict : bool (default=False)
+        print processing steps during predict
+
+    Notes
+    -----
+    Only ``futr_exog_list`` will be considered as exogenous variables.
+
+    References
+    ----------
+    .. [1] https://pandas.pydata.org/pandas-docs/stable/user_guide/timeseries.html#offset-aliases
+    """  # noqa: E501
+
+    _tags = {
+        # packaging info
+        # --------------
+        "authors": ["yarnabrina"],
+        "maintainers": ["yarnabrina"],
+        "python_version": ">=3.8",
+        "python_dependencies": ["neuralforecast"],
+        # estimator type
+        # --------------
+        "y_inner_mtype": "pd.Series",
+        "X_inner_mtype": "pd.DataFrame",
+        "scitype:y": "univariate",
+        "requires-fh-in-fit": True,
+        "X-y-must-have-same-index": True,
+        "handles-missing-data": False,
+        "capability:insample": False,
+    }
+
+    def __init__(
+        self: "_NeuralForecastAdapter",
+        freq: str,
+        local_scaler_type: typing.Optional[
+            typing.Literal["standard", "robust", "robust-iqr", "minmax"]
```

Comment:
```suggestion
            typing.Literal["standard", "robust", "robust-iqr", "minmax", "boxcox"]
```

Reply:
Thanks, added. It's not showing here for some reason.

https://nixtlaverse.nixtla.io/neuralforecast/core.html#neuralforecast

Reply:
We removed it in a previous PR but [this one](https://github.com/Nixtla/neuralforecast/pull/896) adds it back.